.mk/test.mk

```diff
@@ -13,12 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# exclude auto-generated DB code as well as mocks
-# in future, we may want to parse these from a file instead of hardcoding them
-# in the Makefile
-COVERAGE_EXCLUSIONS="internal/db\|/mock/"
-COVERAGE_PACKAGES=./internal/...,./cmd/...
-
```

Comment:
Why did we remove these?

Reply:
Because go 1.22 crashed with that commit